server/app/services/agent/node_plugger.rb

```diff
@@ -18,6 +18,7 @@ def plugin!
       begin
         prev_seen_at = node.last_seen_at
         self.update_node
+        self.publish_update_event
```

Comment:
If I'm interpreting the mongoid docs correctly, then `node.update_attributes(...)` should result in exactly the same atomic mongo `collections[...].find(...).update($set => ...)` operation, with the difference of also triggering the mongoid hooks?

It's not really that significant, I'm mostly just curious as to if I'm understanding the Mongoid behavior here correctly.